sdk/cosmos/ci.yml

```diff
@@ -101,6 +105,10 @@ parameters:
     displayName: 'azure-cosmos-test'
     type: boolean
     default: true
+  - name: release_azurecosmoskafka
```

Comment:
This might not work as we don't release kafka connect on maven, better to comment it out for now so we can unblock this PR.

Reply:
Update, discussed with @FabianMeiswinkel offline, and we should publish it to maven